doc/GUIDE.md

```diff
@@ -835,6 +843,30 @@ can also specify additional clang compiler options, by creating a file named
 `compile_flags.txt` in the current folder, as described
 [here](https://clang.llvm.org/docs/JSONCompilationDatabase.html#alternatives).
 
+## Unsupported Features
+
+There are several elements for which jextract can not generate bindings:
+
+- Function-like macros (as mentioned in the [section on constants](#constants-macros--enums))
+- Bit fields. You will see a warning about bit fields being skipped, such as:
+  
+  ```txt
+  WARNING: Skipping Foo.x (bitfields are not supported)
+  ```
+
+- Opaque types. When a struct or union type is declared but not defined, like:
+
+  ```c
+  struct Foo;
+  ```
+
+  Jextract is not able to generate the regular `Foo` class. You will see a warning about
+  these structs being skipped, such as:
+
+  ```txt
+  WARNING: Skipping Foo (type Declared(Foo) is not supported)
+  ```
+
```

Comment:
Bitfields missing?

Reply:
Bitfields is there. I'll add an item for types > 64 bits

Reply:
I see bitfields now, but we should say something for e.g. `long double` on Linux